styles/index.scss

```diff
@@ -625,6 +638,46 @@ button.UNX-page-next {
   font-weight: 600;
   line-height: 24px;
 }
+.UNX-facets-all {
+  transition: all .3s ease-in;
+}
+.UNX-accordian-close .UNX-facets-all {
+  max-height: 0px;
+  overflow: hidden;
+  opacity: 0;
+}
+.UNX-facets-item.UNX-view-more {
+  height: 100px;
+}
+.UNX-facets-item {
+  height: auto;
+  overflow:scroll;
+}
+.UNX-accordian-open .UNX-facets-all {
+  max-height: 300px;
+  opacity: 1;
+  overflow: scroll;
+}
+.view-more-row {
+  display: flex;
+  justify-content: flex-end;
+}
+.view-more-row button {
+  background: transparent;
+  border-color: transparent;
+  color: #071fd8;
+}
+.range-facets-block .UNX-facets-item {
```

Comment:
any reason `view-more-row` & `range-facets-block` classes are not prefixed with `UNX-` ?